pkg/logs/internal/launchers/container/tailerfactory/file.go

```diff
@@ -141,8 +141,15 @@ func (tf *factory) makeDockerFileSource(source *sources.LogSource) (*sources.Log
 func (tf *factory) findDockerLogPath(containerID string) string {
 	switch runtime.GOOS {
 	case "windows":
+		// if the user has set a custom docker data root, this will pick it up
+		// and set it in place of the usual docker base path
+		dockerLogsPath := dockerLogsBasePathNix
```

Comment:
```suggestion
		dockerLogsPath := dockerLogsBasePathWin
```
Otherwise you will use the *nix path for windows